WordPress/gutenberg
# Font Library: add e2e tests for Upload tab

## What?

Adds some e2e tests to cover the changes made in https://github.com/WordPress/gutenberg/pull/59332.

## How?

Tests the following scenarios:

- When user has permissions to edit font families
  - should display the "Upload" tab
- When user does not have permission to edit font families
  - should not display the "Upload" tab

## Testing Instructions




Run the following command to run these tests locally and ensure they pass: `npm run test:e2e:playwright -- test/e2e/specs/site-editor/font-library.spec.js`
